# Values are in pixels
gui_elements:
  element_names:
    - e1
    - e2
    - e3
    - e4
  e1:
    center:
      x: 1000
      y: 100
    width: 200
    height: 100
  e2:
    center:
      x: 250
      y: 250
    width: 300
    height: 100
  e3:
    center:
      x: 1500
      y: 1000
    width: 300
    height: 100
  e4:
    center:
      x: 400
      y: 1000
    width: 300
    height: 100
